python/mxnet/gluon/utils.py

```diff
@@ -61,13 +52,13 @@ def split_data(data, num_slice, batch_axis=0, even_split=True):
     size = data.shape[batch_axis]
     if size < num_slice:
         raise ValueError(
-            "Too many slices for data with shape %s. Arguments are " \
-            "num_slice=%d and batch_axis=%d."%(str(data.shape), num_slice, batch_axis))
+            "Too many slices for data with shape %s. Arguments are "
```

Comment:
Isn't the backslash required?

Reply:
Isn't triple quotes useful in this case? """blahs dah"""

Reply:
I think triplequotes would be the equiv of

```
"Too many slices for data with shape %s. Arguments are \n" \
"num_slice=%d and batch_axis=%d."%(str(data.shape), num_slice, batch_axis))
```

but I'll try a few options.